batch/manage-pool/manage-pool-windows.sh

```diff
@@ -25,7 +48,13 @@ az batch pool create \
     --vm-size small \
     --start-task-command-line "cmd /c dir /s" \
     --start-task-wait-for-success \
-    --application-package-references myapp
+    --application-package-references myapp # ?? must exist?
```

Comment:
The value passed to the parameter **application-package-references** has to be the name of the Application followed by '#VersionNumber'. For example "MyApplication#1".